.gitignore

```diff
@@ -17,3 +17,6 @@ xcuserdata
 
 # Cocoapods
 Pods
+
+# Mac
+**/.DS_Store
```

Comment:
Please move this to a global gitignore on your computer. It's far easier than adding it to every repo.

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/